target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -10,6 +10,9 @@ case "$FIRMWARE" in
 "ath11k/IPQ8074/hw2.0/cal-ahb-c000000.wifi.bin")
 	case "$board" in
 	arcadyan,aw1000|\
+		askey,rt5010w-d187)
```

Comment:
Fix the whitespace here, and this shouldnt be here since you are changing the caldata extraction of another device

Reply:
move this to the the prpl,haze section, no need to create duplicate of the same command

Reply:
fixed